ci-operator/config/openshift/kubernetes/openshift-kubernetes-master__okd-scos.yaml

```diff
@@ -49,18 +67,94 @@ images:
   to: pod
 promotion:
   to:
-  - name: scos-4.16
+  - additional_images:
+      artifacts: artifacts
+    excluded_images:
+    - base-with-rpms
+    name: scos-4.16
     namespace: origin
+raw_steps:
+- pipeline_image_cache_step:
+    commands: |
+      mkdir -p _output/local/releases;
+      touch _output/local/releases/CHECKSUM;
+      echo $'FROM bin-cross AS bin-cross\nFROM rpms AS rpms\nFROM centos:stream9 \nCOPY --from=bin-cross /go/src/k8s.io/kubernetes/_output/local/releases /srv/zips/ \nCOPY --from=oc-rpms /go/src/github.com/openshift/oc/_rpmbuild/SOURCES/* /srv/zips/ \nCOPY --from=rpms /go/src/k8s.io/kubernetes/_output/local/releases/rpms/* /srv/repo/ \nCOPY --from=oc-rpms /go/src/github.com/openshift/oc/_rpmbuild/RPMS/x86_64/* /srv/repo/ \nCOPY --from=cloud-provider-aws-rpms /go/src/k8s.io/cloud-provider-aws/_output/RPMS/x86_64/* /srv/repo/ \nCOPY --from=cloud-provider-azure-rpms /go/src/github.com/kubernetes-sigs/cloud-provider-azure/_output/RPMS/x86_64/* /srv/repo/ \nCOPY --from=cloud-provider-gcp-rpms /go/src/k8s.io/cloud-provider-gcp/_output/RPMS/x86_64/* /srv/repo/ ' > _output/local/releases/Dockerfile;
+      openshift-hack/build-go.sh;
+    from: bin
+    to: bin-cross
+- project_directory_image_build_step:
+    from: origin_scos-4.16_base-stream9
+    inputs:
+      bin-cross:
+        as:
+        - bin-cross
+        paths:
+        - destination_dir: .
+          source_path: /go/src/k8s.io/kubernetes/_output/local/releases/Dockerfile
+      cloud-provider-aws-rpms:
+        as:
+        - cloud-provider-aws-rpms
+      cloud-provider-azure-rpms:
+        as:
+        - cloud-provider-azure-rpms
+      cloud-provider-gcp-rpms:
+        as:
+        - cloud-provider-gcp-rpms
+      oc-rpms:
+        as:
+        - oc-rpms
+      rpms:
+        as:
+        - rpms
+      src: {}
+    optional: true
+    to: artifacts
+- output_image_tag_step:
+    from: artifacts
+    optional: true
+    to:
+      name: stable
+      namespace: ""
+      tag: artifacts
+- rpm_image_injection_step:
+    from: origin_scos-4.16_base-stream9
+    to: base-with-rpms
 releases:
+  initial:
+    integration:
+      name: scos-4.16
+      namespace: origin
   latest:
     integration:
+      include_built_images: true
       name: scos-4.16
       namespace: origin
 resources:
   '*':
     requests:
       cpu: 100m
-      memory: 200Mi
+      memory: 4Gi
+  artifacts:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  bin:
+    requests:
+      cpu: "3"
+      memory: 7Gi
+  bin-cross:
+    requests:
+      cpu: "3"
+      memory: 8Gi
```

Comment:
the resource requests seem a bit excessive to me, however this was copied from ocp master configs.